### PR TITLE
First simple version of `flutter attach`.

### PR DESCRIPTION
**Pull request by floitschG**

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/attach.dart [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```
Comment (mravn-google) [inline packages/flutter_tools/lib/src/commands/attach.dart]: 2018 now

File: packages/flutter_tools/lib/executable.dart [hunk 2]
```diff
@@ -48,6 +49,7 @@ Future<Null> main(List<String> args) async {
 
   await runner.run(args, <FlutterCommand>[
     new AnalyzeCommand(verboseHelp: verboseHelp),
+    new AttachCommand(),
```
Comment (mravn-google) [inline packages/flutter_tools/lib/executable.dart]: We should pass in `verboseHelp`

File: packages/flutter_tools/lib/src/commands/attach.dart [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    final Device device = await findTargetDevice();
+    final FlutterDevice flutterDevice =
+        new FlutterDevice(device, trackWidgetCreation: false);
+    flutterDevice.observatoryUris = [
+      Uri.parse('http://$ipv4Loopback:$observatoryPort/')
+    ]; // observatoryUris;
+    final HotRunner hotRunner = new HotRunner(
+      <FlutterDevice>[flutterDevice],
+      debuggingOptions: new DebuggingOptions.enabled(getBuildInfo()),
```
Comment (mravn-google) [inline packages/flutter_tools/lib/src/commands/attach.dart]: Not really clear to me what the `BuildInfo` is used for in this context.

**Review by mravn-google — COMMENTED** [review]
Just some superficial feedback, haven't tried to use this yet.

**Comment by Hixie** [comment]
this looks great, thanks for doing this!

we should add a device lab integration test for this

cc @tvolkert

**Comment by floitschG** [comment]
Somehow made this PR unusable.
Creating a new one...

**Pull request closed without merging**